src/components/Hero/Hero.module.scss

```diff
@@ -0,0 +1,157 @@
+.container {
+    position: relative;
+    height: 699px;
+    padding: 15px;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+    background-image: url('../../images/header-bg.png');
+}
+
+.logo {
+    position: absolute;
+    margin: auto;
+    margin-top: -72px;
+    top: 0; 
+    left: 0; 
+    right: 0;
+
+    background-image: url('../../images/logo.png');
+    background-repeat: no-repeat;
+    width: 150px;
+    height: 150px; 
+}
+
+.vertical {
+    display: inline-block;
+    color: #ffffff;
+    writing-mode: vertical-rl;
+    transform: rotate(-180deg);
+    margin-left: 81px;
+    margin-top: 150px;
+}
+
+.heroContainer {   
+    position: absolute;
+    max-width: 998px;
+    margin: 0;
+ 
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+}
+
+.title {
+    font-family: Banny-Round;
+    font-size: 96px;
+    line-height: 0px;
+
+    align-items: center;
+    text-align: center;
+    text-transform: uppercase;
+
+    color: #FFFFFF;
+    margin-bottom: 60px;
+}
+
+.titleMain {
+display: block;
+margin-bottom: 60px;
+font-family: Open Sans;
+font-style: normal;
+font-weight: bold;
+font-size: 12px;
+line-height: 14px;
+
+align-items: center;
+text-align: center;
+letter-spacing: 0.5em;
+text-transform: uppercase;
+
+color: #FFFFFF;
+}
+
+.line {
+    border: none; 
+    background-color: #E8C300; 
+    color: #E8C300;
+    width: 60px;
+    height: 5px; 
+}
+
+.buttonWrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
+
+.addBtn {
+width: 200px;
+height: 60px;
+margin-top: 60px;
+margin-right: 40px; 
+color: #fff; 
+background: transparent;
+border-radius: 5px;
+cursor: pointer;
+overflow: hidden;
+outline: none;
+
+align-items: center;
+text-align: center;
+text-transform: uppercase;
+
+}
+
+.addBtn:last-child {
+    margin-right: 0;
+}
+
+.addBtn:hover, :focus {
+background: #E8C300;
```

Comment:
it is better to format something like this part of code, using nesting. For example: 
.addBtn {
  width: 200px;
  height: 60px;
  ...
*empty line*
  &:last-child {
      margin-right: 0;
  }
  *empty line*
  &:hover, &:focus {
      ...
  }
}